Snowflake.Data.Tests/UnitTests/SFDbCommandTest.cs

```diff
@@ -34,6 +36,33 @@ public void TestCommandWithConnectionAndCommandText()
             Assert.AreEqual(commandText, command.CommandText);
         }
 
+        [Test]
+        public void TestCommandExecuteThrowsExceptionWhenCommandTextIsNotSet()
+        {
+            // Arrange
+            SnowflakeDbConnection conn = new SnowflakeDbConnection();
+
+            // Act
+            var thrown = Assert.Throws<Exception>(() => command.ExecuteScalar());
+
+            // Assert
+            Assert.AreEqual(thrown.Message, "Unable to execute command due to command text not being set");
+        }
+
+        [Test]
+        public void TestCommandExecuteAsyncThrowsExceptionWhenCommandTextIsNotSet()
+        {
+            // Arrange
+            SnowflakeDbConnection conn = new SnowflakeDbConnection();
```

Comment:
the conn variable is not used, and the test passes without it

Reply:
Removed